### PR TITLE
MINOR: enable consumer auth tests

### PR DESCRIPTION
**Pull request by lianetm**
No changes in logic, just enabling some consumer auth tests that were marked as blocked on an open consumer bug, but they are not really affected by it. All tests pass on trunk.

### CODE REVIEW

**Comment by lianetm** [comment]
Hey @chia7712, would you have time for a quick look at this minor follow-up for https://github.com/apache/kafka/pull/17670? These tests were tagged as blocked on KAFKA_17696 but they aren't really, they just pass on trunk (the ones that are truly blocked on KAFKA_17696 will be enabled as part of it). Thanks!

File: core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala [hunk 1]
```diff
@@ -1346,16 +1346,16 @@ class AuthorizerIntegrationTest extends AbstractAuthorizerIntegrationTest {
   }
 
   @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
-  @MethodSource(Array("getTestQuorumAndGroupProtocolParametersClassicGroupProtocolOnly_KAFKA_17696"))
-  def testOffsetFetchWithNoTopicAccess(quorum: String, groupProtocol: String): Unit = {
+  @MethodSource(Array("getTestQuorumAndGroupProtocolParametersAll"))
+  def getTestQuorumAndGroupProtocolParametersClassicGroupProtocolOnly_KAFKA_17696(quorum: String, groupProtocol: String): Unit = {
```
Comment (chia7712) [inline core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala]: typo?

**Review by chia7712 — APPROVED** [review]
LGTM except for one small typo

File: core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala (hunk 1 above)
Comment (lianetm) [reply core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala]: oh yes, my bad, fixed.

**Pull request merged**